stablehlo/dialect/StablehloAttrs.td

```diff
@@ -109,8 +109,9 @@ def StableHLO_OutputOperandAlias : AttrDef<StableHLO_Dialect, "OutputOperandAlia
 }
 
 // Represents a unique identifier for each Send/Recv instruction pair or
-// optionally for collective instructions (AllReduce, CollectivePermute,
-// AllToAll). Non-positive channel_id handle is equivalent to no channel id.
+// optionally for collective instructions (AllToAll, AllReduce, 
```

Comment:
Minor change: `(AllReduce, AllToAll, CollectiveBroadcast, and CollectivePermute)`

Reply:
Done